### DISCUSSION TITLE
some example discussion

### DISCUSSION THREAD

**Discussion by reinerh** [rusty-snake/zufall#2] [Q&A] [answered]
here is some example text that can be discussed about.

**Comment by reinerh** [comment]
and this is a reply to the top level post.

**Reply by reinerh** [reply]
and here is a reply to a reply.

**Reply by reinerh** [reply]
another reply?

**Comment by reinerh** [comment, marked as answer]
this will be the answer to your question.

**Reply by reinerh** [reply]
interesting that the reply marked as "answer" does not appear at the top...

**Reply by rusty-snake** [reply]
You can sort: Oldest | Newest | Top

and top means most up votes or answered?

**Reply by reinerh** [reply]
hm ok, i see. but i would have expected that the marked answer is pinned at the top (by default) or so, so others can locate it easier.

**Reply by rusty-snake** [reply]
> and top means most up votes or answered?

It's by votes. So the answer with the most community support gets up rather then the answer accepted by OP.
